src/collection-resource.js

```diff
@@ -85,6 +86,12 @@ class CollectionResource extends Resource {
       .then(paginateCollectionsProductConnectionsAndResolve(this.graphQLClient));
   }
 
+  fetchWithProductsForCollectionView(id, {productsFirst = 20} = {}) {
```

Comment:
Does this mean the solution we're implementing for WhyFiles will be limited to 20 products? IF so, I think we'll need to up this to 40 at the minimum for right now as their collection _currently_ has 36 items